shared/src/commonMain/kotlin/org/hyperskill/app/request_review/modal/presentation/RequestReviewModalFeature.kt

```diff
@@ -0,0 +1,45 @@
+package org.hyperskill.app.request_review.modal.presentation
+
+import org.hyperskill.app.analytic.domain.model.AnalyticEvent
+
+object RequestReviewModalFeature {
+    internal sealed interface State {
+        object Awaiting : State
+        object Negative : State
+        object Positive : State
+    }
+
+    data class ViewState(
+        val title: String,
+        val description: String?,
+        val positiveButtonText: String,
+        val negativeButtonText: String
+    )
```

Comment:
Why do we need description, positiveButtonText and negativeButtonText in the viewState? These data are static, they are not changed. In my opinion, it is redundant and, we can just directly use string resources on the ui side.

Reply:
@XanderZhu please take a look at Figma, all data dynamic